### ISSUE TITLE
aria-live="assertive" is not read in Edge browser after child is added

### ISSUE THREAD

**Issue by taylortdixon** [nvaccess/nvda#8023]
When a child is added to a div with the attribute `aria-live="assertive"`, Microsoft Edge is not reading this appended child. In addition, I am looking for any guidance on how to get around this for the Edge browser. 

### Steps to reproduce:
>
> Example:
>
> 1. Open Edge browser
> 2. Browse to https://codepen.io/anon/pen/QQroZz
> 3. Click "Add Error" button
> 4. Notice an error "Error in Here" is added to the #errors div.

### Expected behavior:
> NVDA should read "Error in Here" after it's added

### Actual behavior:
> NVDA does not read "Error in Here" in Edge browser. "Error in Here" is read in latest Chrome and Firefox browsers

### System configuration:
NVDA version:
> 2017.4

NVDA Installed or portable:
> Portable

Windows version:
> Microsoft Windows 10 Pro 10.0.14393 Build 14393

Name and version of other software in use when reproducing the issue:
> Microsoft Edge 38.14393.2068.0
### Other questions:

Does the issue still occur after restarting your PC?
Yes

**Comment by taylortdixon** [comment]
Verified this behavior still exists in 2018.1

**Comment by chinanderm** [comment]
I'm using aria-live="polite" but I am also experiencing this with my web applications.

**Comment by mxcosu** [comment]
This is still an issue with Edge - tested with both Jaws and NVDA

**Comment by aloeugene** [comment]
I can say it's still an issue too. Tested with Edge + NVDA 2019.2.

**Comment by DanilRostov** [comment]
Still an issue. Tested with NVDA 2019.2

**Comment by drazen-j** [comment]
Still an issue with NVDA

**Comment by Adriani90** [comment]
This issue does not occur in Edge Cromium 81. I am closing this as works for me since Edge Cromium will be distributed to most users through Windows 10 20 H1 update.

**Comment by Adriani90** [comment]
Adding appropriate label since this is probably still reproducible in Edge Spartan and UIA related environments.
